backend/src/v1/case_file/case_file.service.ts

```diff
@@ -427,20 +427,25 @@ export class CaseFileService {
       query: `mutation createDecision($input: CreateDecisionInput!) {
         createDecision(input: $input) {
           caseIdentifier
+          decision { id }
         }
       }`,
       variables: { input: model },
     });
 
     const returnValue = await this.handleAPIResponse(result);
-    return returnValue?.createDecison;
+    // return returnValue?.createDecison;
```

Comment:
Removing this commented out code will remove the sonar cloud warning

Reply:
this was just to help figure out why the request kept failing on me, and is cleaned up now